system/kube-system/charts/ingress/templates/ingress-service.yaml

```diff
@@ -2,8 +2,10 @@ kind: Service
 apiVersion: v1
 
 metadata:
-  name: ingress-controller 
-
+  name: ingress-controller
+  annotations:
+      prometheus.io/scrape: "true"
+      prometheus.io/port: "9101"
```

Comment:
@MaikuMori: We are experimenting with a custom `nginx.tpl` that contains instrumentation using https://github.com/knyar/nginx-lua-prometheus providing requests metrics.

This is not in a public repository yet so this is confusing. Apologies for that.

@auhlig Maybe we should switch the service annotation to the metrics port of the nginx-controller and add another annotation to the pod template for our custom metrics.